core/src/main/kotlin/com/malinskiy/marathon/execution/strategy/impl/batching/FixedSizeBatchingStrategy.kt

```diff
@@ -1,11 +1,12 @@
 package com.malinskiy.marathon.execution.strategy.impl.batching
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import com.malinskiy.marathon.execution.strategy.BatchingStrategy
 import com.malinskiy.marathon.test.Test
 import com.malinskiy.marathon.test.TestBatch
 import java.util.*
 
-class FixedSizeBatchingStrategy(private val size: Int) : BatchingStrategy {
+data class FixedSizeBatchingStrategy(@JsonProperty("size") private val size: Int) : BatchingStrategy {
```

Comment:
Why data class?

Reply:
Because it implements the equals and hashcode. I can implement it manually